src/main/java/subway/line/domain/Sections.java

```diff
@@ -0,0 +1,48 @@
+package subway.line.domain;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Embeddable
+public class Sections {
+    @OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL, orphanRemoval = true)
+    @JoinColumn(name = "line_id")
+    private List<Section> sections = new ArrayList<>();
+
+    public void add(final Section section) {
+        sections.add(section);
+    }
+
+    private Long getLastDownStationId() {
+        return sections.get(sections.size() - 1).getDownStationId();
+    }
+
+    public boolean isUpStationAlreadyExists(Long stationId) {
+        return sections.stream()
+                .map(Section::getUpStationId)
+                .anyMatch(id -> id.equals(stationId));
+    }
+
+    public List<Long> getStationIds() {
+        List<Long> stationIds = sections.stream()
+                .map(Section::getUpStationId)
+                .collect(Collectors.toList());
+
+        stationIds.add(getLastDownStationId());
+        return stationIds;
+    }
+
+    public boolean isLastStation(final Long stationId) {
```

Comment:
```suggestion
    private boolean isLastStation(final Long stationId) {
```

private로 접근제어를 하는게 좋지 않을까요~?

Reply:
엇 이부분은 Line 클래스에서 쓰고있습니다 ㅠ
혹시 이부분 피드백이 Line에 있는 add와 remove의 책임을 Sections에 옮겨보라는 의견이실까요?

Reply:
엇! 제가 잘못봤네요! 요건 그대로 하셔도 될 것 같습니다.